examples/2_read_serial.py

```diff
@@ -29,6 +29,15 @@
         len(i.particles)))
     for ps in i.particles:
         print("\t {0}".format(ps))
+        print("With records:")
+        for r in i.particles[ps]:
+            print("\t {0}".format(r))
+
+    # printing a scalar value
+    s_e=[ps for ps in i.particles][0]
+    s_c=[r for r in i.particles[s_e]][0]
+    charge=i.particles[s_e][s_c][openpmd_api.Mesh_Record_Component.SCALAR][0]
+    print("And ", s_c, " = ", charge)
```

Comment:
```suggestion
    print("And first particle in particle species {0} has {1} = {2}"
          .format(s_e, s_c, charge))
```

Reply:
This is really a cool way to print it. Thanks for this correction @ax3l !